Game/Source/GDKTestGyms/GDKTestGymsGameInstance.h

```diff
@@ -16,6 +16,10 @@ class GDKTESTGYMS_API UGDKTestGymsGameInstance : public USpatialGameInstance
 
 	bool Tick(float DeltaSeconds);
 	virtual void OnStart() override;
+
+	UFUNCTION()
```

Comment:
Does this need to be a UFUNCTION? I thought the delegate used a member function ptr to bind UObject delegates (I don't know though)

Reply:
Oh I thought it did. Can check.